templates/table_view.html

```diff
@@ -9,5 +9,16 @@
 </head>
 <body>
   {{.table_name}}
+  <form action="/add-column" method="post">
+    <label for="table_name">&#43; Add Column</label>
+    <input type="hidden" id="table_name" name="table_name" placeholder="table-name" value="{{.table_name}}"/>
+    <input type="text" id="column_name" name="column_name" placeholder="column-name"/>
+    <label for="data_type"></label>
+    <select name="column_type">
+      <option value="CHARACTER(20)">CHARACTER(20)</option>
```

Comment:
can you add more datatypes like int etc?

Reply:
Sure! Added integer, float and boolean data types 🥷